common/Log.cpp

```diff
@@ -104,6 +104,85 @@ namespace Log
         }
     };
 
+    class BufferedConsoleChannel : public ConsoleChannel
+    {
+    protected:
+        std::size_t size() const { return _size; }
+        std::size_t available() const { return BufferSize - _size; }
+
+        inline void flush()
+        {
+            if (_size)
+            {
+                write(_buffer, _size);
```

Comment:
Short writes ? =) EINTR ?

Reply:
Yes, internal. You found the syscall with EINTR retry.

Reply:
An old C programmer would appreciate a different name to the std C write function ;-)

Reply:
Renamed.